features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -116,7 +116,7 @@ namespace ExtendedMaterials
 			wsum += weights[i];
 		}
 		float invwsum = rcp(wsum);
-		[unroll] for (int i = 0; i < 6; i++)
+		[unroll] for (i = 0; i < 6; i++)
```

Comment:
is this legal?

Reply:
If it is cpp it'll be out of scope, but idk if hlsl does anything funky.

Reply:
yah hlsl didn't like it, hence the warning. I'm not seeing any issues because of it but we can monitor